Makefile

```diff
@@ -290,13 +290,17 @@ ${ISTIO_OUT}/$(1):
 endef
 $(foreach ITEM,$(PILOT_GO_BINS_SHORT),$(eval $(call pilotbuild,$(ITEM))))
 
+.PHONY: istioctl
+istioctl:
```

Comment:
<pre>- istioctl: 
+ istioctl ${ISTIO_OUT}/istioctl: 
</pre>
which is basically a short-form of what "pilotbuild" is trying to do (and could have been implemented as):
<pre>
$(1) ${ISTIO_OUT}/$(1):
 	bin/gobuild.sh ${ISTIO_OUT}/$(1) istio.io/istio/pkg/version ./pilot/cmd/$(1)
</pre>